app/views/shared/_sidebar_image.html.erb

```diff
@@ -1,6 +1,3 @@
 <% if image %>
-  <figure class="sidebar-image">
-    <img src="<%= image["url"] %>" alt="<%= image["alt_text"] %>">
-    <% if image["caption"].present? %><figcaption><%= image["caption"] %></figcaption><% end %>
-  </figure>
+  <%= render 'components/figure', src: image["url"], alt_text: image["alt_text"], caption: image["caption"] %>
```

Comment:
Can we just use 'alt' rather than 'alt_text'? This would mirror the dom api closer.